.github/actions/create-deployment-package/action.yml

```diff
@@ -0,0 +1,31 @@
+name: "Create deployment package"
+description: "Create a deployment package for Headless and Atomic"
+runs:
+  using: composite
+  steps:
+    - id: parse-version
+      uses: ./.github/actions/parse-version
```

Comment:
I created this JS action to parse the package version and access it in other steps.

Reply:
I think `@action/core` is pretty much OK.